src/frontends/tensorflow_common/src/op/reduce.cpp

```diff
@@ -31,6 +31,29 @@ OutputVector translate_direct_reduce_op(const NodeContext& node) {
     return {reduce_op};
 }
 
+OutputVector translate_prod_op(const NodeContext& node) {
```

Comment:
please use this translator in `op_table` and define in `common_table` header as for other translators